### PR TITLE
Remove legacy json-bulk-access api implementation

### PR DESCRIPTION
**Pull request by nkoester**

### CODE REVIEW

File: rest-api-model-server/src/main/kotlin/org/modelix/sample/restapimodelserver/Serialization.kt [hunk 1]
```diff
@@ -12,11 +13,14 @@ private val logger = LoggerFactory.getLogger("Serialization")
...
-    lectureRef = this.unwrap().reference.serialize(),
+    lectureRef = this.typedReference().ref.serialize(),
     name = this.name,
     description = this.description,
     maxParticipants = this.maxParticipants,
-    room = this.raw_room?.reference?.serialize() ?: ""
+    // TODO: this is a workaround for MODELIX-423
+    // https://issues.modelix.org/issue/MODELIX-423/Serialization-of-references-do-not-add-prefix
+    room = ""
+    // room = this.raw_room?.reference?.serialize() ?: ""
```
[the remark this answers is not in the document]
Comment (languitar) [reply rest-api-model-server/src/main/kotlin/org/modelix/sample/restapimodelserver/Serialization.kt:23]: Any chance to still fix this? Doesn't look nice to leave this in here.

[the remark this answers is not in the document]
Comment (languitar) [reply rest-api-model-server/src/main/kotlin/org/modelix/sample/restapimodelserver/Serialization.kt:23]: What's the verdict here?

[the remark this answers is not in the document]
Comment (nkoester) [reply rest-api-model-server/src/main/kotlin/org/modelix/sample/restapimodelserver/Serialization.kt:23]: there are fixes in the new release which will solve the issue. waiting for this to get rid of all these workarounds

File: mps/solutions/University.Schedule.mps.backend.sandbox/models/University.Schedule.mps.backend.sandbox.mps [hunk 2]
```diff
@@ -42,7 +42,7 @@
   <node concept="1dfXjb" id="3_cs9tOt5VE">
     <node concept="1dfXj3" id="3_cs9tOt5VF" role="1dfXja">
       <property role="TrG5h" value="Einstein" />
-      <property role="1dfXj0" value="69" />
+      <property role="1dfXj0" value="100" />
```
[the remark this answers is not in the document]
Comment (nkoester) [reply mps/solutions/University.Schedule.mps.backend.sandbox/models/University.Schedule.mps.backend.sandbox.mps:45]: I entered some more sensible values.

**Pull request merged**